<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.acams.ddf.testcases.Cms_Eligibility_Test" tests="1" time="117.267" ignored="0" failures="1" timestamp="22 Dec 2017 09:53:26 GMT" skipped="0" hostname="AcroLap15" errors="0">
  <testcase name="Cms_Eligibility_Test" time="117.267" classname="com.acams.ddf.testcases.Cms_Eligibility_Test">
    <failure type="java.lang.AssertionError" message="Alert Text not Matches--Actual Alert Text =The Eligibility End Date field is mandatory. Please enter Eligibility End Date.--Expected Alert Text=Record saved successfully.">
      <![CDATA[java.lang.AssertionError: Alert Text not Matches--Actual Alert Text =The Eligibility End Date field is mandatory. Please enter Eligibility End Date.--Expected Alert Text=Record saved successfully.
	at org.testng.Assert.fail(Assert.java:94)
	at com.acams.ddf.base.BaseTest.reportFailure(BaseTest.java:702)
	at com.acams.ddf.base.BaseTest.verifyAlertPresentAndAlertText(BaseTest.java:945)
	at com.acams.ddf.testcases.Cms_Eligibility_Test.Cms_Eligibility_Test(Cms_Eligibility_Test.java:282)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Cms_Eligibility_Test -->
</testsuite> <!-- com.acams.ddf.testcases.Cms_Eligibility_Test -->
